### ISSUE TITLE
[Accessibility]A11y_VSCode_VSCAdditionalTasks_Screenreader: Jaws does not read the control type of check box.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51855]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details:**
JAWS 18.0.5038 ,MAS1.3.1

**Repro Steps:**
1.On Visual Studio Code Start Menu Folder screen,click on "Next" button.
2.Navigate to check box present on the screen.
3.Check using tab key whether Jaws reads the control types of check box or not

**Actual:**
Jaws does not read the control type of check box. Its reads only name & checked & unchecked.

**Expected:**
Jaws should read the control types of checkbox.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will not able to get information about control.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

Does this issue occur when all extensions are disabled?: Yes

**Attachment for Reference:**
[[Accessibility]A11y_VSCode_VSCAdditionalTasks_Screenreader.pptx](https://github.com/Microsoft/vscode/files/2101250/Accessibility.A11y_VSCode_VSCAdditionalTasks_Screenreader.pptx)

**Comment by joaomoreno** [comment]
@AccessibilityTestingTeam-TCS This needs to go into https://github.com/jrsoftware/issrc
